src/Components/Homepage.js

```diff
@@ -6,9 +6,14 @@ const Homepage = () => {
   const [search, setSearch] = useState("");
   const [filterQuery, setFilterQuery] = useState("");
 const getCountries = async () => {
-  const res = await fetch("data.json")
-  const data = await res.json()
-  setCountries(data)
+   try {
+      const res = await fetch("https://restcountries.com/v3.1/all");
+      const data = await res.json();
+      setCountries(data);
+    } catch (error) {
+      console.log("Error fetching countries:", error);
```

Comment:
try to set error to the screen instead of logging to console

Reply:
will do that